Server.Core/Response.cs

```diff
@@ -6,7 +6,13 @@ public class Response
     {
         public int StatusCode { get; set; }
         public string ContentType { get; set; }
-        public MemoryStream Body { get; set; }
+        public Stream Body { get; set; }
+
+        public void SetBody(byte[] bodyMessage)
```

Comment:
Why have this method on `Response` but a similar method on `RequestBuilder`? There seems to be some missing symmetry.